packages/react/src/Card/Card.features.stories.tsx

```diff
@@ -148,7 +150,7 @@ WithIconSVG.storyName = 'Icon (native)'
 export const IconAndLabel: StoryFn<typeof Card> = () => {
   return (
     <Card href="https://github.com">
-      <Card.Icon icon={<GitBranchIcon />} color="purple" hasBackground />
+      <Card.Icon icon={GitBranchIcon} color="purple" hasBackground />
```

Comment:
FYI there's a noticeable visual difference now, with the overall icon dimensions being smaller. I _think_ it's a bit cleaner now tbh, but wanted to check this is something Site intended for? cc. @danielguillan

https://github.com/user-attachments/assets/d14b296c-9e53-477f-a17d-254b7a9d49f6

Reply:
Related, the reduced spacing here is different to how we show the same UI in the README project pages. E.g. 

<img width="245" alt="Screenshot 2024-10-29 at 12 27 36" src="https://github.com/user-attachments/assets/1293ac9e-0df0-4ee1-a790-16fda563ed3f">

Reply:
Defer to @danielguillan on this. The behaviour in the PR matches the Figma, but I'd also like confirmation from Dani that we're happy with this change.

Reply:
I agree, this doesn't look quite right. We should go with `12px` paddings instead of the current `8px` on sizes `20` and `24` on the Icon component. [Updated Figma](https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=15444-214&node-type=frame&t=JKBjUZCPyhUCNrQv-11)

Reply:
Thanks @danielguillan — that's updated and I've regenerated the snapshots, let me know what you both think 
👍